definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -119,19 +128,7 @@ declare class moment$MomentDuration {
 declare class moment$Moment {
   static ISO_8601: string;
   static (string?: ?string): moment$Moment;
-  static (
-    initDate:
-      | moment$MomentOptions
-      | number
-      | Date
-      | Array<number>
-      | moment$Moment
-      | string
-      | null
-      | void
-      | []
-      | {}
-  ): moment$Moment;
```

Comment:
Could you please provide links to documentation and such to be able to assert this change?